src/App.js

```diff
@@ -1,119 +1,61 @@
-import { useState } from "react";
-import EditPokemon from "./Components/EditPokemon/EditPokemon";
-import AddPokemon from "./Components/AddPokemon/AddPokemon";
-import Table from "./Components/Table/Table";
-import Collections from "./Components/Collections/Collections";
-import Filter from "./Components/Filter/Filter";
-import CreateUser from "./Components/CreateUser/CreateUser";
-
-import {
-    useAllPokemonCards,
-    useAllSets,
-    useAllTypes,
-    useAllCollections,
-} from "./Helpers/apiCalls";
+import React, { useState } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Header from "./Components/Header/Header";
+import CollectionsPage from "./Pages/CollectionsPage";
+import Home from "./Pages/Home";
+import PageNotFound from "./Pages/PageNotFound";
+import { useAllSets, useAllTypes, useAllCollections } from "./Helpers/apiCalls";
 
 function App() {
-    //state variables
-
-    const [selectedPokemon, setSelectedPokemon] = useState(0);
-    const [pokemonCardData, setPokemonCardData] = useState({});
+    //react query variables
+    const { data: sets, isSuccess: setsSuccess } = useAllSets();
```

Comment:
you may want to bring these down lower to be with the other React Query for collections